### ISSUE TITLE
JsDoc new line asterisk not recognized as punctuation

### ISSUE THREAD

**Issue by icetbr** [microsoft/TypeScript-TmLanguage#910]
```js
/**      punctuation.definition.comment.js
 *       comment.block.documentation.js
 */      punctuation.definition.comment.js
```

If not desired to change the base language, would this fit in a grammar extension? I would like to apply different color to the punctuation and the comment itself.

Thanks

**Comment by iDad5** [comment]
Have you tried grammar injection?

**Comment by icetbr** [comment]
No, because I don't like needles! :-) Well, seems like an interesting idea, when I get back to this I will try it.

**Comment by iDad5** [comment]
@icetbr I have to admit, that I don’t really get what you refer to by ‚needle‘ I guess RegEx though. 
Your issue has reminded me that I don’t like the situation you refer to in my slowly developing theme and I will try to find a solution and share it if you like.
